lib/arel_extensions/date_duration.rb

```diff
@@ -40,8 +40,8 @@ def second
       ArelExtensions::Nodes::Duration.new "s", self
     end
 
-    def format(tpl)
-      ArelExtensions::Nodes::Format.new [self, tpl]
+    def format(tpl, time_zone = nil)
+      ArelExtensions::Nodes::Format.new [self, tpl, Arel::Nodes.build_quoted(time_zone)]
```

Comment:
I don't think you want to `build_quoted` nil.  I was thinking of
```
ArelExtensions::Nodes::Format.new [self, tpl, (Arel::Nodes.build_quoted(time_zone) if time_zone)]
```
Also, I would probably have put this at the lowest level (the ctor).  That's more consistent with the handling of the other args, and seems somewhat more natural.